views/default/forms/admin/site/icons.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+$site_icon = elgg_format_element('div', ['class' => 'elgg-subtext'], elgg_echo('admin:site_icons:info'));
```

Comment:
why in a `elgg-subtext`? Why not in full text `output/longtext`?

Reply:
changed to longtext